hack/test-cmd.sh

```diff
@@ -257,6 +251,12 @@ echo "config files: ok"
 export KUBECONFIG="${HOME}/.kube/non-default-config"
 export CLUSTER_ADMIN_CONTEXT=$(oc config view --flatten -o template -t '{{index . "current-context"}}')
 
+pushd "$OS_ROOT" > /dev/null
+restore_nullglob=$(shopt -p nullglob)
```

Comment:
I would posit you could reduce that million years to under a minute if you availed yourself of a manpage or a terminal window.  At the first least, the proposed change is less deceptively straightforward than `find -print0 | sort -u | xargs -0 -n1 printf "%s\n"`, which apparently got through review despite two defects, never mind the excessive use of non-builtins.  Better to require the Bash novice to RTFM to look up that `shopt` command than to deceive Bash novices and more experienced people too (evidently) into a false sense of comprehension. 

Would a comment on the Bash code suffice, or would you prefer I fix the broken `find | sort | xargs printf` command so that it does what it looks like it does?